diff --git a/vllm/third_party/flash_linear_attention/ops/chunk_o.py b/vllm/third_party/flash_linear_attention/ops/chunk_o.py
index 0c323b8..8506b52 100644
--- a/vllm/third_party/flash_linear_attention/ops/chunk_o.py
+++ b/vllm/third_party/flash_linear_attention/ops/chunk_o.py
@@ -16,7 +16,7 @@ from vllm.triton_utils import tl, triton

 from .index import prepare_chunk_indices
 from .op import exp
-from .utils import FLA_CHUNK_SIZE, check_shared_mem, is_nvidia_hopper
+from .utils import FLA_CHUNK_SIZE, check_shared_mem, is_amd, is_nvidia_hopper

 BKV_LIST = [64, 128] if check_shared_mem() else [32, 64]
 NUM_WARPS = [2, 4] if is_nvidia_hopper else [2, 4, 8]
@@ -31,10 +31,10 @@ NUM_WARPS = [2, 4] if is_nvidia_hopper else [2, 4, 8]
 @triton.autotune(
     configs=[
         triton.Config({"BK": BK, "BV": BV}, num_warps=num_warps, num_stages=num_stages)
-        for BK in BKV_LIST
-        for BV in BKV_LIST
+        for BK in ([32] if is_amd else BKV_LIST)
+        for BV in ([32] if is_amd else BKV_LIST)
         for num_warps in NUM_WARPS
-        for num_stages in [2, 3, 4]
+        for num_stages in ([2] if is_amd else [2, 3, 4])
     ],
     key=["H", "K", "V", "BT"],
 )
